src/lib/viz/style/default-styles.ts

```diff
@@ -2,14 +2,14 @@ const NULL_COLOR = '#ccc';
 const OTHERS_COLOR = '#777';
 const NULL_SIZE = 0;
 const STROKE_WIDTH = 0.5;
-const STROKE_COLOR = '#22222288';
+const STROKE_COLOR = '#22222244';
 
 export const defaultStyles: any = {
   Point: {
     color: '#EE4D5A',
-    size: 10,
+    size: 8,
     opacity: 1,
-    strokeColor: STROKE_COLOR,
+    strokeColor: '#FFF',
```

Comment:
My proposal, feel free to push it up as the new STROKE_COLOR if it fits

Reply:
I think the objective is to fit with CARTOframes style helpers, not with VL ones, right?

Reply:
And overall, especially on style helpers, they match and they are great. But in this particular  case of just defaults, I think that a grey border is clearly worse than a white one for points, as it doesn't allow to see clearly the circles. Honestly I'm a bit surprised that they use that in CF. We can discuss a bit with them if we need to

Reply:
Helpers were born in CF, so it's natural to me to take them as a reference, but IMO defaults should work in a broader context

Reply:
In any case, don't take me too seriously on this. If they are pretty sure about that default, we can just respect it and move on. In that case, just revert my commit and let's move on

Reply:
As we have talked before, we are going to use a white stroke. I am going to add a commit with that.